src/runtime/Converter.cs

```diff
@@ -782,7 +782,7 @@ internal static bool ToPrimitive(BorrowedReference value, Type obType, out objec
                             goto type_error;
                         }
                         double num = Runtime.PyFloat_AsDouble(value);
-                        if (num == -1.0 && Exceptions.ErrorOccurred())
```

Comment:
Why remove the `num` check? The doc says that is the value returned on failure, and checking for it saves us a PInvoke trip in `ErrorOcurred`.

Reply:
Good point, I just wanted to simplify it, but the previous variant is better.